File: 110algoritmos/exercicios013/ex013-Python/ex013.py
# Faça um algoritmo que leia o salário de um funcionário e mostre o seu novo
# salário, com 15% de aumento. entrada: 4319.43 saida: 4967.34
sal = float(input("Qual é o salário do Funcionário? R$"))
aumento = sal * 1.15
print(f"Um funcionário que ganhava R${sal}, com 15% de aumento, passa a receber R${aumento:.2f}")

File: 110algoritmos/exercicios013/ex013-Go/ex013.go
package main

import "fmt"

func main() {
	var sal float32

	fmt.Print("Qual é o salário do funcionário? R$")
	fmt.Scan(&sal)
	aumento := sal * 1.15
	fmt.Printf("\nUm funcionário que ganhava R$%v, com 15%% de aumento, passa a ganhar R$%.2f", sal, aumento)

}
